tests/end_to_end/repo/.github/workflows/ci.yml

```diff
@@ -48,4 +48,4 @@ jobs:
         if: steps.coverage_comment.outputs.COMMENT_FILE_WRITTEN == 'true'
         with:
           name: python-coverage-comment-action
-          path: python-coverage-comment-action.txt
+          path: python-coverage-comment-action*.txt
```

Comment:
Note for self: check if we should do the same change in the README examples, if not already done.

Reply:
No we're good, it was already part of the documentation.